stable/nfs-server-provisioner/Chart.yaml

```diff
@@ -2,12 +2,13 @@ apiVersion: v1
 appVersion: 1.0.8
 description: nfs-server-provisioner is an out-of-tree dynamic provisioner for Kubernetes. You can use it to quickly & easily deploy shared storage that works almost anywhere.
 name: nfs-server-provisioner
-version: 0.1.0
+version: 0.1.1
 maintainers:
 - name: kiall
   email: kiall@macinnes.ie
+home: https://github.com/kubernetes/charts/tree/master/stable/nfs-server-provisioner
 sources:
-- https://github.com/kubernetes-incubator/external-storage/tree/master/nfs
+- https://github.com/kubernetes/charts/tree/master/stable/nfs-server-provisioner
```

Comment:
can you keep the previous source, this is usually used for linking to Dockerfiles and other sources and not really this repo

Reply:
Ah, yes. I miss read the url

Reply:
fixed :)